_posts/2019-07-24-cockpit-199.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 199
+author: kkoukiou
+date: 2019-07-24
+tags: cockpit linux
+slug: cockpit-199
+category: release
+summary: Cockpit with Logs and Services and HWInfo design updates
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 199.
+
+### Redesigned logs
+
+Restyled logs across all of Cockpit.
+
+![service-logs](/images/service-logs.png)
+
+### Services: Design and accesibility improvements
+
+The enablement and runtime status of units is now represented in a clearer way.
+
+![service-details](/images/service-details.png)
+
+In addition the tabs in Service page are now accessible.
+
+### System: Show DIMM information on Hardware Info page
+
+The Hardware Info page now can show information about DIMM memory, such as size, rank, speed etc.
+
+![dimm-memory](/images/dimm-hwinfo.png)
+
+_Thanks to Akshit Dua for this feature!_
+
+### Machines: VM creation dialog now shows the recommended memory for the selected OS
+
+The VM creation dialog will now validate the memory field in order to be above
+the allowed minimum and will also show the recommended value for the selected OS.
```

Comment:
Ugh, another broken line comment. We really should not hard wrap in Markdown files. It makes editing them more difficult. Generally, relying on an editor to wrap lines is the way to go for written text, and is what Markdown, AsciiDoc, and other formats recommend.

Anyway:

```suggestion
During VM creation, memory fields are checked for validity. The Create VM dialog now displays a recommended RAM size for the selected OS and an error appears if memory is set too low.
```